[pkg/stanza/fileconsumer] Failing test TestExpectCallsTimeout

### Component(s)

pkg/stanza/fileconsumer

### Describe the issue you're reporting

[Failing test link](https://github.com/open-telemetry/opentelemetry-collector-contrib/actions/runs/14909815354/job/41881137599?pr=39940)

Failure output:
```
=== Failed
ERROR rerun aborted because previous run had a suspected panic and some test may not have run
=== FAIL: fileconsumer/internal/emittest TestExpectCallsTimeout (0.01s)
    sink.go:137: 
        	Error Trace:	/home/runner/work/opentelemetry-collector-contrib/opentelemetry-collector-contrib/pkg/stanza/fileconsumer/internal/emittest/sink.go:137
        	            				/home/runner/work/opentelemetry-collector-contrib/opentelemetry-collector-contrib/pkg/stanza/fileconsumer/internal/emittest/sink_test.go:163
        	Error:      	timeout: expected: 10, actual: 0
        	Test:       	TestExpectCallsTimeout
panic: test executed panic(nil) or runtime.Goexit

goroutine 20 [running]:
testing.tRunner.func1.2({0x6d6720, 0x9014f0})
	/opt/hostedtoolcache/go/1.23.8/x64/src/testing/testing.go:1632 +0x3fc
testing.tRunner.func1()
	/opt/hostedtoolcache/go/1.23.8/x64/src/testing/testing.go:1635 +0x6b6
runtime.Goexit()
	/opt/hostedtoolcache/go/1.23.8/x64/src/runtime/panic.go:635 +0x5e
testing.(*common).FailNow(0xc0001364e0)
	/opt/hostedtoolcache/go/1.23.8/x64/src/testing/testing.go:1006 +0x85
github.com/stretchr/testify/require.ElementsMatch({0x77ea40, 0xc0001364e0}, {0x6c8b00, 0xc0001bdde8}, {0x6c8b00, 0xc0001bde00}, {0x0, 0x0, 0x0})
	/home/runner/go/pkg/mod/github.com/stretchr/testify@v1.10.0/require/require.go:102 +0x107
github.com/open-telemetry/opentelemetry-collector-contrib/pkg/stanza/fileconsumer/internal/emittest.(*Sink).ExpectCalls(0xc000114900, 0xc0001364e0, {0xc000146460, 0x1, 0x1})
	/home/runner/work/opentelemetry-collector-contrib/opentelemetry-collector-contrib/pkg/stanza/fileconsumer/internal/emittest/sink.go:141 +0x425
github.com/open-telemetry/opentelemetry-collector-contrib/pkg/stanza/fileconsumer/internal/emittest.TestExpectCallsTimeout(0xc000136340)
	/home/runner/work/opentelemetry-collector-contrib/opentelemetry-collector-contrib/pkg/stanza/fileconsumer/internal/emittest/sink_test.go:167 +0x479
testing.tRunner(0xc000136340, 0x73e380)
	/opt/hostedtoolcache/go/1.23.8/x64/src/testing/testing.go:1690 +0x227
created by testing.(*T).Run in goroutine 1
	/opt/hostedtoolcache/go/1.23.8/x64/src/testing/testing.go:1743 +0x826

```

--- a/pkg/stanza/fileconsumer/internal/emittest/sink_test.go
+++ b/pkg/stanza/fileconsumer/internal/emittest/sink_test.go
@@ -24,7 +24,7 @@ func TestNextToken(t *testing.T) {
 }
 
 func TestNextTokenTimeout(t *testing.T) {
-	s, testCalls := sinkTest(t, WithTimeout(10*time.Millisecond))
+	s, testCalls := sinkTest(t, WithTimeout(30*time.Millisecond))
 	for _, c := range testCalls {
 		token := s.NextToken(t)
 		assert.Equal(t, c.Body, token)
@@ -46,7 +46,7 @@ func TestNextTokens(t *testing.T) {
 }
 
 func TestNextTokensTimeout(t *testing.T) {
-	s, testCalls := sinkTest(t, WithTimeout(10*time.Millisecond))
+	s, testCalls := sinkTest(t, WithTimeout(30*time.Millisecond))
 	for i := 0; i < 5; i++ {
 		tokens := s.NextTokens(t, 2)
 		assert.Equal(t, testCalls[2*i].Body, tokens[0])
@@ -69,7 +69,7 @@ func TestNextCall(t *testing.T) {
 }
 
 func TestNextCallTimeout(t *testing.T) {
-	s, testCalls := sinkTest(t, WithTimeout(10*time.Millisecond))
+	s, testCalls := sinkTest(t, WithTimeout(30*time.Millisecond))
 	for _, c := range testCalls {
 		token, attributes := s.NextCall(t)
 		require.Equal(t, c.Body, token)
@@ -90,7 +90,7 @@ func TestExpectToken(t *testing.T) {
 }
 
 func TestExpectTokenTimeout(t *testing.T) {
-	s, testCalls := sinkTest(t, WithTimeout(10*time.Millisecond))
+	s, testCalls := sinkTest(t, WithTimeout(30*time.Millisecond))
 	for _, c := range testCalls {
 		s.ExpectToken(t, c.Body)
 	}
@@ -109,7 +109,7 @@ func TestExpectTokens(t *testing.T) {
 }
 
 func TestExpectTokensTimeout(t *testing.T) {
-	s, testCalls := sinkTest(t, WithTimeout(10*time.Millisecond))
+	s, testCalls := sinkTest(t, WithTimeout(30*time.Millisecond))
 	for i := 0; i < 5; i++ {
 		s.ExpectTokens(t, testCalls[2*i].Body, testCalls[2*i+1].Body)
 	}
@@ -128,7 +128,7 @@ func TestExpectCall(t *testing.T) {
 }
 
 func TestExpectCallTimeout(t *testing.T) {
-	s, testCalls := sinkTest(t, WithTimeout(10*time.Millisecond))
+	s, testCalls := sinkTest(t, WithTimeout(30*time.Millisecond))
 	for _, c := range testCalls {
 		s.ExpectCall(t, c.Body, c.Attributes)
 	}
@@ -152,7 +152,7 @@ func TestExpectCalls(t *testing.T) {
 }
 
 func TestExpectCallsTimeout(t *testing.T) {
-	s, testCalls := sinkTest(t, WithTimeout(10*time.Millisecond))
+	s, testCalls := sinkTest(t, WithTimeout(30*time.Millisecond))
 	testCallsOutOfOrder := make([]emit.Token, 0, 10)
 	for i := 0; i < len(testCalls); i += 2 {
 		testCallsOutOfOrder = append(testCallsOutOfOrder, testCalls[i])

